keps/sig-storage/1790-recover-resize-failure/kep.yaml

```diff
@@ -21,11 +21,12 @@ see-also:
 replaces:
 superseded-by:
 
-latest-milestone: "v1.31"
+latest-milestone: "v1.32"
 stage: "alpha"
 
 milestone:
   alpha: "v1.23"
+  beta: "v1.32"
```

Comment:
Do you plan to request PRR by this PR? If so, you need to update /keps/prod-readiness/sig-storage/1790.yaml

Reply:
I think it is already updated -  https://github.com/kubernetes/enhancements/blob/master/keps/prod-readiness/sig-storage/1790.yaml